opensearch/src/main/java/org/opensearch/sql/opensearch/setting/OpenSearchSettings.java

```diff
@@ -6,8 +6,7 @@
 package org.opensearch.sql.opensearch.setting;
 
 import static org.opensearch.common.settings.Settings.EMPTY;
-import static org.opensearch.common.unit.TimeValue.timeValueDays;
-import static org.opensearch.common.unit.TimeValue.timeValueMinutes;
+import static org.opensearch.common.unit.TimeValue.*;
```

Comment:
nit: rest of the team doesn't like wildcard imports, this was probably done automatically by the IDE. [You can disable it.]